setup.py

```diff
@@ -105,6 +105,7 @@
 TESTS_REQUIRE = [
     # test dependencies
     "absl-py",
+    "nltk",  # for NIST and probably others
```

Comment:
something must have already installed it since the tests worked until now, but ok to have it explicitely :)